crates/ruff_python_formatter/src/expression/expr_tuple.rs

```diff
@@ -135,6 +152,10 @@ impl NeedsParentheses for ExprTuple {
         source: &str,
         comments: &Comments,
     ) -> Parentheses {
+        // We use Custom as signal for RemoveUnlessBreaks
+        if parenthesize == Parenthesize::RemoveUnlessBreaks {
```

Comment:
Should the parentheses also be removed if the node has leading comments?

Reply:
in the case where we use this (`for (a, b) in x`) there shouldn't be any leading comments for the tuple expression